### DISCUSSION TITLE
如何测试单向带宽

### DISCUSSION THREAD

**Discussion by ClemenceCc** [baidu/dperf#415] [General]
在FAQ中，17项说了one-way，场景是只发送，不响应。
如何配置，可以使，客户端只发送，服务端只接受。

出现这种场景是，测试千兆带宽时候，双向测试，bitsRx和bitsTx都是512Mbps，所以想要尝试，如何实现单边达到1024Mbps

**Comment by pengjianzhang** [comment]
cliet上设置flood，向80端口发送报文
server上配置监听81端口，或者不开启server
